### ISSUE TITLE
Mentions of quote-shares

### ISSUE THREAD

**Issue by Liwott** [friendica/friendica#11441]
### Is the feature request related to a problem? Please describe.

Quote-sharing is a well working and good looking way of including citations to other posts and comments. However, I am thinking it is a bit of a pity that no mention of that appears on the shared post. 

This would allow to follow the future directions taken by original ideas on the long run, similarly to what one can do with scientific publications (cf [my related forum post](https://nerdica.net/display/ec054ce7-1962-6652-0932-45d016786914))

### Describe the feature you'd like

A "cited by" list of links to all quotes of a posts.

### Describe alternatives you've considered

An easy workaround is to explicitly add a comment on the original thread. So the addressed problem is far from being a liability, but to have an automatic mechanism for this would be a nice improvement.

### Additional context

[According to Michael Vogel](https://pirati.ca/display/ec054ce7-1962-6652-0932-45d016786914), the possibility of implementing that features is currently held back by improvements yet to be made on the implementation of sharing.

**Comment by MrPetovan** [comment]
I disagree with the core concept of this feature. If you want to be featured in the conversation, post a comment to it, and you can include quotes of the original post in it. It will show to your followers as well and you will be part of the conversation.

Quote Share is meant to start a parallel discussion the original author may not want to be aware about unless they also follow you.

**Comment by annando** [comment]
I think that is a good thing to get notified about a quote share of one of your posts. Quote shares can be used for good and bad. Being noticed does help in keeping an eye on the discussions.

**Comment by MrPetovan** [comment]
I don't think it has anything to do with good or bad behavior. But since you're bringing it up, I don't think you want to be notified about the bad quotes, but there's no way to distinguish both. The only thing we can distinguish is the difference between a comment on the original conversation and a quote share, and I believe they should have a different outcome as well.

Adding a comment for quote shares, while technically possible, blurs the difference between both features and questions the very existence of the quote share feature.

**Comment by Liwott** [comment]
Thank you both for your replies!

> the original author may not want to be aware about unless they also follow you.

There could be a setting to opt out of getting a notification when it happens. But I don't see a disadvantage of having the mention appear on the original comment. The interest of getting such notifications for those who want to is also a way to guarantee one's right of reply.

> Adding a comment for quote shares, while technically possible, blurs the difference between both features and questions the very existence of the quote share feature.

If fact commenting under the reshared post is the current workaround, and the feature I was proposing is in fact a different kind of feedback that would contribute to further distinguish the two features.

Having also in mind the possibility of the notification being turned of, this would also avoid to spam with an actual comment the user uninterested in knowing about the reshare.

What I have in mind is something similar to the "... reshared this" message, except that it would be for quote-shares, and hence should also link the repost.

**Comment by MrPetovan** [comment]
I don't mind it if it's passive like the current "X reshared this" that doesn't trigger a notification and only shows about your contacts, but a comment on the original post isn't the right way to do this.
